gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/longrunning/stub/HttpJsonOperationsStub.java

```diff
@@ -309,32 +325,113 @@ protected HttpJsonOperationsStub(
     backgroundResources = new BackgroundResourceAggregation(clientContext.getBackgroundResources());
   }
 
-  private static <RequestT extends Message, ResponseT>
-      ApiMethodDescriptor<RequestT, ResponseT> getApiVersionedMethodDescriptor(
-          ApiMethodDescriptor<RequestT, ResponseT> methodDescriptor, String apiVersion) {
-    if (apiVersion == null) {
-      return methodDescriptor;
+  /* OperationsClient's RPCs are mapped to GET/POST/DELETE and this function only expects those HttpVerbs to be used */
+  private static String getValueBasedOnPatternCase(HttpRule httpRule) {
```

Comment:
This method does not have to be `static` now.

Reply:
Done